tests/css_comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -96,14 +96,26 @@ exports[`selector.css 1`] = `
 .powerPathNavigator table.powerPathInfo th:active,
 .powerPathNavigator table.powerPathInfo th:active + th:last-child {
 }
+
+/* pressed buttons */
+.powerPathNavigator .helm button.pressedButton,
+.powerPathNavigator .helm button:active:not(.disabledButton) {
+}
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .powerPathNavigator .helm button.pressedButton,
 /* pressed buttons */
-.powerPathNavigator .helm button:active:not(.disabledButton),
+  .powerPathNavigator
+  .helm
+  button:active:not(.disabledButton),
```

Comment:
See my comment somewhere else about comments in CSS. We should extract comments inside of CSS via likely a custom tokenizer and feed it through prettier's comments handling logic. This would solve most of the issues we have with comments in CSS.